src/Sylius/Bundle/ApiBundle/CommandHandler/Account/VerifyCustomerAccountHandler.php

```diff
@@ -44,6 +52,14 @@ public function __invoke(VerifyCustomerAccount $command): JsonResponse
         $user->setEmailVerificationToken(null);
         $user->enable();
 
+        $channel = $this->channelContext->getChannel();
+        $localeCode = $this->localeContext->getLocaleCode();
```

Comment:
IMO this handler should not use contexts to get these data, it should be provided by the command.